docs/docs/pair-programming.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Open Community Pair Programming Sessions
+---
+
+The best part of open source is the community, and every community is stronger when it works together. To help build the strongest community possible, Gatsby is offering free [one-hour pair programming sessions][cal] to anyone and everyone in the open source community.
+
+## How community pair programming sessions work
+
+1.  Sign up for any available slot on [the open pairing calendar][cal]
+2.  You’ll be paired with one of the Gatsby Inkteam
+3.  Join the video meeting during your time slot
+4.  Build, learn, or discuss your Gatsby or other OSS project!
+
+### What we expect in pair programming sessions
+
+- Open community pair programming sessions are _only_ for Gatsby and/or open source projects
```

Comment:
My question here is what other kinds of open source projects would you be open to pairing on projects that are open source but don't involve / aren't related to Gatsby? This makes it sound like you would

Reply:
Personally, I think we should help with pretty much anything OSS related. Our goal with this isn't to focus on Gatsby as a codebase, but rather to focus on the open source community at large. We want to get more people involved, and Gatsby may not be the right first step for them. But if we can help get them started now, regardless of the project, hopefully we'll see them contribute to Gatsby in the future.

Reply:
Oh, ok! I hadn't thought of that! Seems cool!